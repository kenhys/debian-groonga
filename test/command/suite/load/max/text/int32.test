table_create Values TABLE_NO_KEY
column_create Values value COLUMN_SCALAR Int32

load --table Values
[
{"value": "2147483647"}
]

select Values --output_columns _id,value
