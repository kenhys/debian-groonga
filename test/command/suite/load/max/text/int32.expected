table_create Values TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Values value COLUMN_SCALAR Int32
[[0,0.0,0.0],true]
load --table Values
[
{"value": "2147483647"}
]
[[0,0.0,0.0],1]
select Values --output_columns _id,value
[[0,0.0,0.0],[[[1],[["_id","UInt32"],["value","Int32"]],[1,2147483647]]]]
